packages/indy-sdk-to-askar-migration/src/IndySdkToAskarMigrationUpdater.ts

```diff
@@ -0,0 +1,363 @@
+import type { AnonCredsCredentialValue } from '@aries-framework/anoncreds'
+import type { Agent, FileSystem } from '@aries-framework/core'
+import type { EntryObject } from '@hyperledger/aries-askar-shared'
+
+import { AnonCredsCredentialRecord, AnonCredsLinkSecretRecord } from '@aries-framework/anoncreds'
+import { AskarWallet } from '@aries-framework/askar'
+import { InjectionSymbols, KeyDerivationMethod, JsonTransformer, TypedArrayEncoder } from '@aries-framework/core'
+import { Key, KeyAlgs, Store, StoreKeyMethod } from '@hyperledger/aries-askar-shared'
+
+import { IndySdkToAskarMigrationError } from './errors/IndySdkToAskarMigrationError'
+import { transformFromRecordTagValues } from './utils'
+
+/**
+ *
+ * Migration class to move a wallet form the indy-sdk structure to the new
+ * askar wallet structure.
+ *
+ * Right now, this is ONLY supported within React Native environments AND only sqlite.
```

Comment:
I think this disclaimer is fine here but should be part of the README. 

I understand the reasons about stating that it works in React Native because it assumes that no issuer or verifier side has been supported there officially, but I don't think it is accurate as we can also have a NodeJS _holder-role_ agent that would be in the same situation.

Also, what about mediation records? Mediator configuration is not taken into account. In that case you assume only current Bifold case where a mediatorInvite is provided at startup and there is apparently no problem on creating the mediation coordination again. But that's not necessarily the case of every app using AFJ.

What about apps using generic records module? And those that have custom plug-ins that store custom records? 🤯 

The latter case is maybe extreme (don't worry for us, as we are starting production using 0.4.0 right away 😎 ), but generic records and maybe proofs are used in some mobile wallets.

Reply:
I think all the cases you described should work in Node.JS, however it's not too much work to add credential definitions (the only one missing) after which it will fully work in Node.JS. We can do that in an upcoming release I think

Reply:
Ah good to see that all records are copied (it´s not always a good idea to review a PR at midnight). So if the only missing objects in migration are the Credential Definitions I think it makes more sense to allow migration in NodeJS as not all server deployments are used to issue credentials (we have a mediator for instance, or other services that verify but not issue credentials).